addons/payment/models/account_move.py

```diff
@@ -21,6 +26,17 @@ def _compute_authorized_transaction_ids(self):
                 lambda tx: tx.state == 'authorized'
             )
 
+    @api.depends('transaction_ids')
+    def _compute_amount_paid(self):
+        """ Sum all the transaction amount for which state is in 'pending', 'authorized' or 'done'
+        """
+        for invoice in self:
+            invoice.amount_paid = sum(
+                invoice.transaction_ids.filtered(
+                    lambda tx: tx.state in ('pending', 'authorized', 'done')
```

Comment:
In fact, after reading through the check in `account_payment`: `and (pending_manual_txs or not tx_ids or invoice.amount_paid &lt; invoice.amount_total)`, you could consider only `authorized`/`done` states, not the pending one (I believe it's kind of confusing to consider a pending tx as paid anyway).
```suggestion
                    lambda tx: tx.state in ('authorized', 'done')
```